Database/customerdb.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+if(fs.existsSync('.env')){
+  require('dotenv').config()
+}
+
+const connectingString = process.env.DATABASE_URL
+
+const pgp = require('pg-promise')()
+
+const db = pgp(connectingString)
+
+const CREATE_CUSTOMER = `INSERT INTO customers VALUES($1,$2)`
+
+const VIEW_CUSTOMER = `READ:(customer_name,paid,style_image,stylist_name,active)
+
```

Comment:
This whitespace confused me; I didn't immediately see the relationship between the text blocks.  Consider using whitespace to make it easier to read the code, keeping logically similar statements together, and providing indentation to help clarify:
```js
const VIEW_CUSTOMER = `
  READ:( customer_name, paid,style_image, stylist_name, active )
  SELECT customers.customer_name, customers.paided, styles.style_image, stylist.name AS stylist_name
  FROM customers
  JOIN appointments ON customers.id = appointments.id
  # etc. 
```